src/quacc/schemas/ase.py

```diff
@@ -349,6 +350,113 @@ def summarize_vib_and_thermo(
     )
 
 
+def summarize_neb_run(
+    dyn: Optimizer,
+    trajectory: Trajectory | list[Atoms] | None = None,
+    charge_and_multiplicity: tuple[int, int] | None = None,
+    additional_fields: dict[str, Any] | None = None,
+    store: Store | None | DefaultSetting = QuaccDefault,
+) -> OptSchema:
+    """
+    Summarize the NEB run results and store them in a database-friendly format.
+
+    Parameters
+    ----------
+    dyn
+        ASE Optimizer object used for the NEB run.
+    trajectory
+        Trajectory of the NEB run, either as a Trajectory object or a list of Atoms objects.
+    charge_and_multiplicity
+        Charge and spin multiplicity of the Atoms object.
+    additional_fields
+        Additional fields to add to the task document.
+    store
+        Maggma Store object to store the results in. Defaults to `QuaccSettings.STORE`.
+
+    Returns
+    -------
+    OptSchema
+        A dictionary containing the summarized NEB run results.
+    """
+    settings = get_settings()
+    store = settings.STORE if store == QuaccDefault else store
+    additional_fields = additional_fields or {}
+
+    # Get trajectory
+    if not trajectory:
+        trajectory = read(dyn.trajectory.filename, index=":")
+
+    n_images = additional_fields["interpolate_flags"]["n_images"]
+
+    n = additional_fields.get("n_iter_return", -1)
+    if n == -1:
+        trajectory = trajectory[-(n_images):]
+    else:
+        trajectory = _get_nth_iteration(
+            trajectory, int(len(trajectory) / n_images), n_images, n
+        )
+    trajectory_results = [atoms.calc.results for atoms in trajectory]
+    ts_index = (
+        np.argmax([i["energy"] for i in trajectory_results[-(n_images - 1) : -1]]) + 1
+    )
+    ts_atoms = trajectory[ts_index]
+
+    for traj_atoms in trajectory:
+        traj_atoms.calc = None
+
+    initial_atoms = trajectory[0]
+
+    base_task_doc = atoms_to_metadata(
+        initial_atoms, charge_and_multiplicity=charge_and_multiplicity
+    )
+
+    # Clean up the opt parameters
+    parameters_opt = dyn.todict()
+    parameters_opt.pop("logfile", None)
+    parameters_opt.pop("restart", None)
+
+    opt_fields = {
+        "highest_e_atoms": ts_atoms,
```

Comment:
Even if the NEB converges, the highest energy image is not necessarily a TS. Technically you always need to do a TSopt on the highest energy image. Perhaps just a matter of semantics...

Reply:
This is true. It would only be a TS with the climbing image variant. 

We can keep it as-is then. The key "highest_e_atoms" is a bit awkward, but I can't really think of a better name to be honest.